src/operator/nn/dnnl/dnnl_pooling-inl.h

```diff
@@ -172,27 +173,34 @@ inline bool DNNLRequireWorkspace(const PoolingParam& param) {
 }
 
 typedef ParamOpSign<PoolingParam> DNNLPoolingSignature;
-void DNNLPoolingCompute(const OpContext& ctx,
-                        const PoolingParam& param,
-                        const NDArray& in_data,
-                        const OpReqType req,
-                        const NDArray& out_data,
-                        const NDArray* workspace,
-                        const bool use_adaptive_pooling);
-
-void DNNLPoolingGradCompute(const OpContext& ctx,
-                            const PoolingParam& param,
-                            const NDArray& out_grad,
-                            const NDArray& in_data,
-                            const NDArray* workspace,
-                            const OpReqType req,
-                            const NDArray& in_grad);
+
```

Comment:
Why there is an empty line here now?

Reply:
Because it is type declaration and for the readability purposes it is in my opinion better for it to be separated. Of course if I'm not missing convention or something